newlib/libc/stdio/open_memstream.c

```diff
@@ -61,7 +61,9 @@ POSIX.1-2008
 Supporting OS subroutines required: <<sbrk>>.
 */
 
-#ifdef __GNUC__
+#include <sys/cdefs.h>
```

Comment:
could just move the pragmas below the long list of other includes?

Reply:
Possibly... unless these pragmas are meant to affect the code inside the headers, which I believe they are in at least one case. I'll do some trial-and-error with the CI system and the compilers I have locally to determine which pragmas can be moved without a detrimental effect.

Reply:
I think it will be necessary to add `#include <sys/cdefs.h>` to `newlib/libm/fenv/fenv.c`. The pragmas there are specifically intended to feed into `#include <fenv.h>`

Reply:
sounds good, thanks for taking a look.